default.nix

```diff
@@ -1,16 +1,32 @@
-with (import (fetchTarball https://github.com/NixOS/nixpkgs/archive/ffea68a09cedab941d19f02c9041689ebc81192e.tar.gz) {});
-# NOTE: when bumping nixpkgs, also update nixpkgs-src.json and .travis.yml
+let
+  # NOTE: when bumping nixpkgs, also update nixpkgs-src.json and .travis.yml
+  pkgs = import (fetchTarball https://github.com/nixos/nixpkgs/archive/fda4b93cd4fd3775408117c380aab0f33737d30f.tar.gz) {};
+  nodejs = pkgs.nodejs-8_x;
+in
 
-stdenv.mkDerivation {
-  name = "daedalus";
-
-  buildInputs = [
-    nix bash binutils coreutils curl
-    git python27 curl electron nodejs-8_x
-    nodePackages.node-gyp nodePackages.node-pre-gyp
-    gnumake
-  ];
+with (import (fetchTarball https://github.com/moretea/yarn2nix/archive/v1.0.0.tar.gz) { inherit pkgs nodejs; });
 
-  src = null;
-
-}
+pkgs.callPackage (
+{ stdenv, python }:
+mkYarnPackage {
```

Comment:
Is "impure development shell" a requirement? e.g. a nix-shell where developers can run yarn, npm, etc.
If so does `mkYarnPackage` support that?

Reply:
@cleverca22 is planning on merging with his branch, that includes a `shell.nix` for this.